### PR TITLE
Add CTRL + ALT + N shortcut for new documents

### PR DESCRIPTION
**Pull request by gokaysatir**
Added CTRL + ALT + N shortcut.
* This sends the NewDocument postmessage to integrator along with the document type.
* Integrator should handle the message.

Change-Id: I8458f382771211b245ad4cedf501b73b50afc1c7

* Resolves: # 
* Target version: master 

### TODO

- [ ] ...

### Checklist

- [ ] Code is properly formatted
- [ ] All commits have Change-Id
- [ ] I have run tests with `make check`
- [ ] I have issued `make run` and manually verified that everything looks okay
- [ ] Documentation (manuals or wiki) has been updated or is not required

### CODE REVIEW

**Comment by eszkadev** [comment]
please fix commit message:
- title is sign off
- 2 change ids

**Comment by gokaysatir** [comment]
> please fix commit message:
> 
> * title is sign off
> * 2 change ids

I changed them thanks :)

File: browser/src/map/handler/Map.Keyboard.js [hunk 1]
```diff
@@ -734,6 +734,13 @@ L.Map.Keyboard = L.Handler.extend({
 			e.preventDefault();
 		}
 
+		// CTRL + ALT + N for new document. This needs to be handled by the integrator.
+		if (this._isCtrlKey(e) && e.altKey && e.key.toUpperCase() === 'N') {
+			this._map.fire('postMessage', {msgId: 'NewDocument', args: { type: this._map.getDocType() }});
```
Comment (eszkadev) [inline browser/src/map/handler/Map.Keyboard.js:739]: It doesn't use standard convention we have for other commands: COOL -> Host UI_xxxx (but there are already 2 exceptions... so not sure :) )

We have already command to create new document: UI_CreateFile
How this is different?
https://sdk.collaboraonline.com/docs/postmessage_api.html#id6

Comment (gokaysatir) [reply browser/src/map/handler/Map.Keyboard.js:739]: Yep, thanks for the catch :) Replaced the command with the existing one.

**Pull request merged**